SU-SWS/decanter-react
# Adapt 3717: Hero Icon updates

# READY FOR REVIEW

# Summary
- Update react-hero-icon package to @heroicon/react tailwind package.

# Needed By (Date)
- tbd

# Urgency
- medium

# Steps to Test

1. Pull this branch
2. Review locally at `/story/simple-hero-icon--default` or see Netlify Preview(s):
- [Hero Icon Component](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-hero-icon--default)
- [Button with icon](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-button--download)
- [Alert](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/composite-alert--default)
- [Dismiss Button](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-dismiss-button--default)
- [Cta Link](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-cta-link--jump)
- [Cta Link (no Icon example)](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-cta-link--no-icon)
- [Cta Button](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/simple-cta-link-button--download)
- [Local Footer](https://deploy-preview-71--decanter-react.netlify.app/?path=/story/stanford-identity-local-footer--button)

# Affected Projects or Products
- DERF

# Associated Issues and/or People
- [ADAPT-3717](https://stanfordits.atlassian.net/browse/ADAPT-3717)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
